debian/control

```diff
@@ -5,10 +5,12 @@ Priority: optional
 Standards-Version: 3.9.8
 Build-Depends:
  asciidoc,
+ clang,
```

Comment:
we weren't compiling the debian package with clang. We could, but that's a totally separated discussion from GIMP, unless GIMP plugin doesn't compile with gcc which would be a problem we need to fix.

Reply:
The plugin builds fine without clang.  Was just wondering, since the docs say clang is recommended.   Will remove.